src/sentry/static/sentry/app/views/organizationGroupDetails/groupSimilarIssues/similarStackTrace/item.tsx

```diff
@@ -21,14 +21,13 @@ import space from 'app/styles/space';
 import {Group, Organization, Project} from 'app/types';
 import {callIfFunction} from 'app/utils/callIfFunction';
 
-const similarInterfaces = ['exception', 'message'];
-
 type Props = {
   issue: Group;
   project: Project;
   orgId: Organization['id'];
   groupId: Group['id'];
   score?: Record<string, any>;
+  v2: boolean;
```

Comment:
```suggestion
  v2: boolean;
  score?: Record<string, any>;
```

we usually add the optional props at the end of the object 😉